[enterprise-4.13] OSDOCS-5611: add ssh key location (#58049)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_cluster-operators-ref[Cluster Cloud Controller Manager Operator] entry in the _Platform Operators reference_.

[discrete]
[id="ocp-4-13-rhcos-ssh-key-location"]
==== Change in SSH key location

> {product-title} {product-version} introduces a {op-system-base} 9.2 based {op-system}. Previously, SSH keys were located in `/home/core/.ssh/authorized_keys` on {op-system}. Now, on {op-system-base} 9.2 based {op-system}, SSH keys are located in `/home/core/.ssh/authorized_keys.d/ignition`.

Reviewer: ```suggestion
{product-title} {product-version} introduces a {op-system-base} 9.2 based {op-system}. Before this update, SSH keys were located in `/home/core/.ssh/authorized_keys` on {op-system}. With this update, on {op-system-base} 9.2 based {op-system}, SSH keys are located in `/home/core/.ssh/authorized_keys.d/ignition`.
```
Hi @jldohmann , consider these suggestions.

Author: Thanks, Tami. I've implemented these suggestions 🙂
